src/content/en/updates/2019/07/chrome-75-media-updates.md

```diff
@@ -0,0 +1,175 @@
+
+The [Media Capabilities API] specification has since been updated to handle
+encrypted media configurations as well so that websites using encrypted media
+(EME) can select the optimal media streams.
+
+In a nutshell, here’s how the Decoding Info API for EME works. Give a try to the
+[official sample].
+
+    const encryptedMediaConfig = {
+      type: 'media-source', // or 'file'
+      video: {
+        contentType: 'video/webm; codecs="vp09.00.10.08"',
+        width: 1920,
+        height: 1080,
+        bitrate: 2646242, // number of bits used to encode a second of video
+        framerate: '25' // number of frames used in one second
+      },
+      keySystemConfiguration: {
+        keySystem: 'com.widevine.alpha',
+        videoRobustness: 'SW_SECURE_DECODE' // Widevine L3
+      }
+    };
+
+    navigator.mediaCapabilities.decodingInfo(encryptedMediaConfig).then(result => {
+      if (!result.supported) {
+        console.log('Argh! This encrypted media configuration is not supported.');
+        return;
+      }
+
+      if (!result.keySystemAccess) {
+        console.log('Argh! Encrypted media support is not available.')
+        return;
+      }
+
+      console.log('This encrypted media configuration is supported.');
+      console.log('Playback should be' +
+          (result.smooth ? '' : ' NOT') + ' smooth and' +
+          (result.powerEfficient ? '' : ' NOT') + ' power efficient.');
+
+      // TODO: Use `result.keySystemAccess.createMediaKeys()` to setup EME playback.
+    });
+
+EME playbacks have specialized decoding and rendering code paths, meaning
+different codec support and performance compared to clear playbacks. Hence a new
+`keySystemConfiguration` key must be set in the media configuration object
+passed to `navigator.mediaCapabilities.decodingInfo()`. The value of this key is
+a dictionary that holds a number of [well-known EME types]. This replicates the
+inputs provided to EME's `requestMediaKeySystemAccess()` (rMKSA) with one major
+difference: sequences of inputs provided to rMKSA are flattened to a single
+value wherever the intent of the sequence was to have rMKSA choose a subset it
+supports.
+
+The Decoding Info API describes the quality (smoothness and power efficiency) of
+support for a single pair of audio and video streams without making a decision
+for the caller. Callers should still order media configurations as they do with
+rMKSA, only now they walk the list themselves.
+
+`navigator.mediaCapabilities.decodingInfo()` returns a promise that resolves
+asynchronously with an object containing three booleans: `supported`, `smooth`,
+and `powerEfficient`. However when a`keySystemConfiguration` key is set and
+`supported` is `true`, yet another MediaKeySystemAccess object named
+`keySystemAccess` is returned as well. It can be used to request some media keys
+and setup encrypted media playback. Here’s an example.
+
+    // Like rMSKA(), orderedMediaConfigs is ordered from most to least wanted.
```

Comment:
Shouldn't this be inside a ```` ```js ... ``` ```` block?

Reply:
It doesn't need to. See https://pr-7920-dot-web-central.appspot.com/web/updates/2019/07/chrome-75-media-updates